src/main/java/org/jenkinsci/plugins/ghprb/extensions/comments/GhprbPublishJenkinsUrl.java

```diff
@@ -33,13 +33,9 @@ public String getPublishedURL() {
     }
 
     public String postBuildComment(AbstractBuild<?, ?> build, TaskListener listener) {
-        StringBuilder msg = new StringBuilder();
-
-        msg.append("\nRefer to this link for build results (access rights to CI server needed): \n");
-        msg.append(generateCustomizedMessage(build));
-        msg.append("\n");
-        
-        return msg.toString();
+        return "\nRefer to this link for build results (access rights to CI server needed): \n" +
```

Comment:
For something this small I'm not sure I agree, but it's just splitting hairs so I'll change it back.